### DISCUSSION TITLE
The Shawshank Redemption was based on a short novel by who?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#419] [Galaxy Trivia] [answered]
**Bonus:** How many years between the novel and the movie?

**Comment by kylekce** [comment, marked as answer]
"The Shawshank Redemption" was based on a short novel titled "Rita Hayworth and Shawshank Redemption" by Stephen King. The novel was published in 1982, while the movie adaptation was released in 1994, making it 12 years between the novel and the movie.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/kylekce?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉

**Comment by peterbaccaro** [comment]
The Shawshank Redemption movie was based on the 1982 **Stephen King** novel called 'Rita Hayworth and Shawshank Redemption'.
The book was first published on the 27th August 1982 and the film was first shown at the Toronto International Film Festival on the 10th September 1994 so there was 12 years and 27 days between the book being first published and the movie being first screened.

**Reply by ParanoidUser** [reply]
Kudos for daily participation and precision! 👏👏

**Reply by peterbaccaro** [reply]
:smile:
